src/actions/index.js

```diff
@@ -0,0 +1,150 @@
+import axios from 'axios';
+import jwt from 'jsonwebtoken';
+
+import instance from './AxiosInstance';
+import { ROOT_URL } from '../constants';
+
+export const AUTHENTICATED = 'authenticated_user';
+export const UNAUTHENTICATED = 'unauthenticated_user';
+export const AUTHENTICATED_ERROR = 'authenticated_error';
+export const REGISTER = 'register_user';
+export const USER_ALREADY_EXISTS = 'user_already_exists';
+export const FETCH_CATEGORIES = 'fetch_categories';
+export const FETCH_RECIPES = 'fetch_recipes';
+export const CREATE_CATEGORY = 'create_category';
```

Comment:
I've made it a habit to put these constants in their own file, as the app grows these constants will become very many and you'll need a way to manage them. Consider seperate files for constants used for different purposes e.g auth, category, general etc